packages/strapi-plugin-users-permissions/controllers/User.js

```diff
@@ -170,10 +170,6 @@ module.exports = {
     const { id } = ctx.params;
     const { email, username, password } = ctx.request.body;
```

Comment:
You Will need to verify that if one of those vars is sent it cannot be empty nor null. You should run the username unique and email unique checks only when a username or an email is provided.

Reply:
Understand :) added these non-empty validation checks now.